src/deepqmc/hkext.py

```diff
@@ -47,12 +48,13 @@ def __init__(
         self,
         in_dim,
         out_dim,
+        residual=False,
+        name=None,
+        *,
         hidden_layers=None,
         bias=True,
         last_linear=False,
         activation=ssp,
-        name=None,
-        residual=False,
         w_init='default',
```

Comment:
Should we remove the defaults here (except for name, which exists only for convinience)?